app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
Got it. Then best to give a description of what it does for you: "Behave! - control the toxic  comments you see.". or "Behave! - choose your threshold for offensive comments.". The words "filter" and "block" are a little misleading. Thoughts?